src/app/[locale]/globals.css

```
@@ -127,3 +143,45 @@ body {
 :is(.dark .theme-icon) {
   filter: invert(1) brightness(100);
 }
+
+/* --- HERO TEXT ANIMATIONS --- */
+@keyframes luxuryReveal {
```

Stylelint (17.13.0)

[error] Expected keyframe name "luxuryReveal" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)